test/Objects/Schema/Generation/SchemaGeneratorTest.php

```diff
@@ -160,6 +168,51 @@ public function it_should_generate_records_containing_records(): void
                         'intType',
                         Schema::int(),
                         Schema\Record\FieldOption::default(42)
+                    )
+                    ->field(
+                        'uuidType',
+                        Schema::uuid()
+                    )
+                    ->field(
+                        'timestampMillisType',
+                        Schema::timestampMillis()
+                    ),
+            )
+            ->field(
+                'unionField',
+                Schema::union(
+                    Schema::null(),
+                    Schema::named('org.acme.SimpleRecord')
+                )
+            );
+
+        $this->assertEquals($expected, $schema);
+    }
+
+    #[Test]
+    public function it_should_generate_records_containing_records_using_attributes(): void
```

Comment:
@mattiabasone Was there a reason that you only implemented this test, and not the other tests?

Reply:
@tPl0ch No reason at all :D I've refactored a bit the test class and fixed an error, what do you think about it?